erpnext/stock/doctype/stock_entry/stock_entry.py

```diff
@@ -1601,3 +1601,51 @@ def validate_sample_quantity(item_code, sample_quantity, qty, batch_no = None):
 			format(max_retain_qty, batch_no, item_code), alert=True)
 		sample_quantity = qty_diff
 	return sample_quantity
+
+def raw_material_update_on_bom():
+	"""
+	Calculates Average Manufactured Qty(avg_manufactured_qty) for BOM on the Basis of Stock Entries created and
+	it fetches the Stock Entries created within past seven days against BOM.
+	"""
+	past_seven_days = get_date_str(add_days(today(), -7))
```

Comment:
Where does it fetch the number of days from settings ?

Reply:
On L1611